base/applications/mspaint/mouse.cpp

```diff
@@ -384,20 +406,30 @@ struct TextTool : ToolBase
     void OnButtonUp(BOOL bLeftButton, LONG x, LONG y)
     {
         imageModel.ResetToPrevious();
-        if (selectionModel.IsSrcRectSizeNonzero())
-        {
-            imageModel.CopyPrevious();
 
-            placeSelWin();
-            selectionWindow.ShowWindow(SW_SHOW);
-            ForceRefreshSelectionContents();
-        }
+        RECT rc;
+        selectionModel.GetRect(&rc);
+
+        INT cxMin = 80, cyMin = 24;
```

Comment:
Maybe these default size values could be `#define`d somewhere (to allow changing the default minimal size), instead of being buried in the code.

Reply:
#define'd `CX_MINTEXTEDIT` and `CY_MINTEXTEDIT` in `textedit.h`.